.github/workflows/ci.yml

```diff
@@ -51,7 +50,7 @@ jobs:
       SECRET_KEY: fake_secret_key_to_run_tests
 
       # This is the URL used by databases on our CI.
-      DATABASE_URL: postgres://postgres:postgres@postgres:5432/postgres
+      DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
```

Comment:
Unsure why we're using localhost here, I'm going to merge this branch into master though so I can unblock getting work onto staging